engine/crates/engine-v2/src/plan/walkers/input_value.rs

```diff
@@ -0,0 +1,133 @@
+use schema::{InputValue, InputValueSerdeError, RawInputValuesContext, SchemaWalker};
```

Comment:
This could benefit some quick unit tests to show what gets deserialized etc.

Reply:
You mean to check serialize, deserialize, etc?
I have some sanity unit-tests here: https://github.com/grafbase/grafbase/blob/81f1bca1bc5c86c07830d8ace8dbaa002612a5bf/engine/crates/engine-v2/schema/src/input_value/raw.rs#L255
in the previous PR